modules/tealium_moments_api/example/lib/main.dart

```diff
@@ -0,0 +1,216 @@
+// ignore_for_file: constant_identifier_names
+
+import 'package:flutter/material.dart';
+import 'dart:async';
+
+import 'package:tealium_moments_api/common.dart';
+import 'package:tealium_moments_api/tealium_moments_api.dart';
+import 'package:tealium/tealium.dart';
+import 'package:tealium/common.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatefulWidget {
+  const MyApp({super.key});
+
+  @override
+  State<MyApp> createState() => _MyAppState();
+}
+
+class _MyAppState extends State<MyApp> {
+  final TextEditingController _momentsApiReferrerController =
+      TextEditingController();
+  final TextEditingController _momentsApiEngineIdController =
+      TextEditingController();
+  String? _momentsApiReferrer;
+  String _momentsApiEngineId = '';
+  MomentsApiRegion _currentRegion = MomentsApiRegion.GERMANY;
+  MomentsApiRegion _momentsApiRegion = MomentsApiRegion.GERMANY;
+
+  var tealiumConfig = TealiumConfig(
```

Comment:
I do have one slightly larger thought - that since the `Engine ID` and region etc are configurable at runtime, but the Tealium `account` and `profile` are not, the "configurable" aspect of the UI is only useful to those with access to the `tealiummobile/demo` account and profile.

fyi @craigrouse - should we allow for users to configure the account/profile in the UI? or assume that if they're running this test app locally, that they are either Tealium employees, or that they have sufficient knowledge to change the account/profile in this code.

Reply:
This is something useful I think but we should've done it on all platforms as well (and I think we didn't?).

Reply:
I'm happy to leave this in the UI, but add a note that says the Engine ID needs to be one that's configured in the account & profile that's hardcoded here. @jacobwthms @jameskeith